bokeh/models/plots.py

```diff
@@ -692,3 +692,13 @@ def webgl(self, webgl):
         When set to ``webgl``, glyphs without a WebGL rendering implementation
         will fall back to rendering onto 2D canvas.
     """)
+
+    aspect = Either(Auto, Float, default=None, help="""
+    Specify the aspect ratio behavior of the plot. Aspect ratio is defined as
+    the physical distance relationship between a figure's width and height. It
+    is defined by the ratio of the width over the height.
+
+    Default is ``None`` which indicates no aspect ratio. ``"auto"`` indicates
+    that the plot should maintain an aspect ratio of 1. A ``Float`` value can
+    be given for arbitrary aspect ratio control.
```

Comment:
We can edit this in a follow-on PR before the release, but this is a little off. `"auto"` indicates that the aspect ratio of the axes dimensions should be automatically made to match the aspect ratio of the corresponding pixel extents of the axes. The end result is that a 1x1 area in data space is a square in pixels, and conversely that a 1x1 pixel is a square in data units, or (perhaps more importantly) that circles drawn inside squares will always touch all four sides. 

We should also add a note about how this setting only affect when there are two dataranges, and then only affects initial plot draw and resets -- it is possible for the use of tools (single axis zoom or unconstrained bo zoom) to change the aspect ratio